src/views/Home.vue

```diff
@@ -53,38 +53,44 @@
             </Block>
           </div>
         </router-link>
+
         <NoResults
           :block="true"
           v-if="Object.keys(spaces).length < 1"
           class="pr-md-4"
         />
       </div>
     </Container>
+    <div id="endofpage" />
   </div>
 </template>
 
 <script>
-import { mapActions } from 'vuex';
 import orderBy from 'lodash/orderBy';
 import spotlight from '@snapshot-labs/snapshot-spaces/spaces/spotlight.json';
-import { infiniteScroll } from '@/helpers/utils';
+import { useScrollMonitor } from '@/composables/useScrollMonitor';
+import { routeState } from '@/composables/useRouter';
+
```

Comment:
Can you make imports with a module appear before imports with `@/`, also can you remove blank line between the imports? (same on others files)